properties.schema

```diff
@@ -11,6 +11,16 @@
       "inputType": "Text",
       "validators": [],
       "translatable": true
+    },
+    "labelLink": {
+      "type": "string",
+      "required": false,
+      "default": "Link to terms that begin with",
```

Comment:
As this ARIA-label is assigned to an `<a>` tag I think you should be fine to do this:
```suggestion
      "default": "Terms beginning with",
```
That way it won't be quite so repetitive when someone's navigating through them with a screenreader